### ISSUE TITLE
`fenced-code-language` (`MD040`) - false positive on `py` codeblock

### ISSUE THREAD

**Issue by DetachHead** [rvben/rumdl#720]
### rumdl version

0.2.31

### Installation method

Other

### Bug description

this rule does not seem to recognize `py` codeblocks, even though they work on github, so i assume that means it's recognized by linguist as an alias for `python`:

```py
def foo():
    ...
```
(you can click "Copy markdown source" on this issue to see that it uses the `py` alias, and syntax highlighting still works)

### Steps to reproduce

1. create a `.md` file with the following content:
   ````md
   ```py
   foo = 1
   ```
   ````
2. run `rumdl check` on it

### Expected behavior

no error

### Actual behavior

```
asdf.py:1:1: [MD040] Unknown language 'py' (not in GitHub Linguist). Syntax highlighting may not work.
```

### Configuration

```toml
[global]
enable = ["MD040"]
[MD040]
unknown-language-action = 'warn'
```

### Operating system

Linux

**Comment by rvben** [comment]
Fixed in v0.2.32. rumdl's Linguist table is generated from a pinned upstream commit, and upstream added `py` (and `py3`) as Python aliases seven weeks after that pin, so the table predated them. Both aliases are now recognized, and a full audit of the table against the pinned commit confirmed no other aliases are missing. Thanks for the report.
